main.go

```diff
@@ -67,6 +61,23 @@ func checkError(e error) {
 	}
 }
 
+// Load the beacon log from file
+func loadLog() BeaconLog {
+	beaconLogData := new(BeaconLog)
```

Comment:
So I've yet to have a need to use `new(T)`. In this case, `var beaconLogData BeaconLog` would make more sense. If you wanted to create a pointer, you could do `beaconLogData := &BeaconLog{}` which would be more common then `new(T)`.